lib/fathead/python/parse.py

```diff
@@ -221,13 +226,19 @@ def create_file(self):
                 if data_element.get('module') or data_element.get('function'):
                     method_signature = data_element.get('method_signature')
                     first_paragraph = data_element.get('first_paragraph')
-                    name = self.create_name_from_data(data_element)
+                    name, redirect = self.create_names_from_data(data_element)
+
+                    # Don't set redirect if name/redirect are the same. Happens when we are only getting a function name
+                    # with no module.
+                    if redirect == name:
+                        redirect = ''
+
                     abstract = '{}{}{}'.format(method_signature, '<br>' if method_signature and first_paragraph else '', first_paragraph)
                     url = data_element.get('url')
                     list_of_data = [
                         name,                       # unique name
                         'A',                        # type is article
-                        '',                         # no redirect data
+                        redirect,                   # no redirect data
```

Comment:
You'll have to change the type to "R" if this is a redirect and leave the other fields blank. So a redirect entry will look like this:
```
module.function    R    function
````